### DISCUSSION TITLE
Cannot import references using 'import' command on windows

### DISCUSSION THREAD

**Discussion by HTony03** [pythonnet/pythonnet#2403] [Q&A]
### Environment

-   Pythonnet version: 3.0.3
-   Python version: 3.10.2
-   Operating System: Windows
-   .NET Runtime: .net  8.0.302

### Details

-   Describe what you were trying to get done.

commands:
```python
import clr
# correctly installed the clr
 clr.AddReference(".\src\RealAirplaneTag")
# returns <System.Reflection.RuntimeAssembly object at 0x00000245F59D5180>
import RealAirplaneTag
"""
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\site-packages\IPython\core\interactiveshell.py", line 3553, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-84-f77e7f71b52a>", line 1, in <cell line: 1>
    import RealAirplaneTag
  File "E:\Program Files\JetBrains\PyCharm Community Edition 2024.1.1\plugins\python-ce\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'RealAirplaneTag'
"""

# also cannot import when using pythonnet.load('coreclr')
```
running platform: python console in pycharm
codebase:
https://github.com/Maicarons/RealAirplaneTag
https://github.com/HTony03/miniairways_mod_manager

the System can be imported :
```python
clr.AddReference('System.IO')
# <System.Reflection.RuntimeAssembly object at 0x000001AC38A23B80>
import System.IO
# normally imported
```

    _TODO_

-   What commands did you run to trigger this issue? If you can provide a
    [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
    this will help us understand the issue.

```python
    print('TODO')
```

-   If there was a crash, please include the traceback here.

```python
    print('TODO')
```

**Comment by DaveSkender** [comment]
We have a working example of how to do this [here](https://github.com/facioquo/stock-indicators-python/blob/a4a39ab515472bd64e7aa2eadcabb25131f9934b/stock_indicators/_cslib/__init__.py#L21-L26), if helpful.

Additionally, our [Stock Indicators for Python Quickstart](https://github.com/facioquo/stock-indicators-python-quickstart) includes detailed step-by-step instructions on how to setup your local environment as well, which is often the problem ... missing the .NET SDK install is a common issue.
